src/components/__internal__/no-choice-list-item.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListOption from '../list-option/list-option.jsx';
+import styles from '../no-choice-list-item.css';
+
+export default function NoChoiceListItem(props) {
```

Comment:
An empty state isn't a list item. It shouldn't be focusable or anything (because otherwise the user might think it is selectable).